src/style.css

```
@@ -1 +1,77 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
```

Move font family to the html selector
```suggestion
html {
  font-family: Avenir, sans-serif;
}
```